src/Stormpath/Oauth/CreateGrantAuthenticationAttempt.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+
+namespace Stormpath\Oauth;
+
+
+use Stormpath\Resource\AccountStore;
+use Stormpath\Resource\Resource;
+
+class CreateGrantAuthenticationAttempt extends Resource
```

Comment:
This will need to remain a resource because it is used in the create method on the datastore to make the post request.  That method requires a Resource to be passed.